spec/features/user_can_navigate_to_specific_page_from_index_page.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.feature "User can navigate to a specific page from the index page" do
+  context "When a page exists" do
+    scenario "the show action page renders when the show link is selected" do
+      test_page = Page.create(title: "test title", content: "test content", slug: "test slug")
```

Comment:
```suggestion
      test_page = Page.create(title: "test title", content: "test content", slug: "test-slug")
```

Slugs shouldn't really contain spaces.

Reply:
The spacing is due to not having the slug validations in place yet. This will be rectified in a future PR